release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1286,6 +1427,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|Hosted control planes for Amazon Web Services (AWS)
```

Comment:
Hi @opayne1! This should be either "Hosted control planes for OpenShift Container Platform" or simply "Hosted control planes".